packages/design-system/src/components/Combobox/Combobox.tsx

```diff
@@ -426,7 +426,7 @@ const ComboboxCreateItem = styled(ComboboxPrimitive.CreateItem)`
   }
   &&:hover,
   &&[data-highlighted] {
-    background-color: transparent;
+    background: ${({ theme }) => theme.colors.neutral0};
   }
```

Comment:
I wonder if you can just remove this block? There's a neutral0 bg defined above already that maybe would be enough. Otherwise this is fine of course

Reply:
If I remove this line I have this result
<img width="710" alt="Screenshot 2025-04-30 alle 11 49 52" src="https://github.com/user-attachments/assets/c63253cd-7ea8-4c27-af00-5810ffcf7539" />
with my solution, instead, we have this background
<img width="703" alt="Screenshot 2025-04-30 alle 11 50 03" src="https://github.com/user-attachments/assets/6fb890fa-424d-4df1-838b-85ef4506f31c" />